src/nvim/charset.h

```diff
@@ -14,8 +14,8 @@
 /// @return Folded variant.
 #define CH_FOLD(c) \
   utf_fold((sizeof(c) == sizeof(char)) \
-             ?((int)(uint8_t)(c)) \
-             :((int)(c)))
+           ? ((int)(uint8_t)(c)) \
```

Comment:
uncrustify modified the `utf_fold` line but didn't fix these lines. Do some of it's rules not work on macros?

Reply:
Honestly, I don't remember exactly what rules I've changes for macros so I can't answer this definitively at the moment.